### DISCUSSION TITLE
You just hit a route that doesn't exist... the sadness.

### DISCUSSION THREAD

**Discussion by tatuananhnb2608** [medusajs/medusa#3765] [Q&A]
You just hit a route that doesn't exist... the sadness.
![image](https://user-images.githubusercontent.com/18524117/230560960-60a73f0a-b303-44d1-a4e0-291a246d139b.png)

**Comment by olivermrbl** [comment]
@tatuananhnb2608 What admin system are you using? Our plugin or the standalone project?

**Reply by tatuananhnb2608** [reply]
I use https://github.com/medusajs/admin and https://github.com/medusajs/nextjs-starter-medusa

**Reply by olivermrbl** [reply]
I will try to dig out the issue.

Though, I highly recommend using our new [admin plugin](https://docs.medusajs.com/admin/quickstart). The project https://github.com/medusajs/admin has entered maintenance mode and will not receive further updates.

**Reply by tatuananhnb2608** [reply]
![image](https://user-images.githubusercontent.com/18524117/230581770-a9c92c7f-aae3-4b1d-8bdf-c2bf19b67c2d.png)
I tried the version you suggested but it gave me an error I had to go find another version to try

**Reply by tatuananhnb2608** [reply]
import {flushSync as $4q5Fq$flushSync} from "/@fs/C:/Users/Admin/shop/admin/node_modules/react-dom/index.js?v=1c6129e8";
I fixed the above error by adding name and "license": "MIT", But new error
![image](https://user-images.githubusercontent.com/18524117/230584746-50eddae4-292a-4469-8f14-53c6083c52a2.png)

**Reply by tatuananhnb2608** [reply]
@olivermrbl Can you help me with the above error?

**Reply by culy247** [reply]
@tatuananhnb2608  if you use admin like separator separator system, i think you need add awilix, iso8601-duration, lodash missing package

**Reply by tatuananhnb2608** [reply]
I have installed awilix, iso8601-duration, lodash but still the same error

**Reply by tatuananhnb2608** [reply]
{
  "name": "admin",
  "license": "MIT",
  "devDependencies": {
    "@medusajs/admin": "^2.0.0"
  },
  "scripts": {
    "build:admin": "medusa-admin build",
    "dev:admin": "medusa-admin dev"
  },
  "dependencies": {
    "awilix": "^8.0.1",
    "iso8601": "^1.1.1",
    "lodash": "^4.17.21",
    "react-dom": "^18.2.0"
  }
}
